content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -159,6 +159,33 @@ dotenv.config() // load the environment variables
 console.log(`The connection URL is ${process.env.DATABASE_URL}`)
 ```
 
+### Use multiple .env files or .env files outside the prisma folder
+
+To use multiple `.env` files:
+
+1.  Install [dotenv-cli](https://www.npmjs.com/package/dotenv-cli):
```

Comment:
I think dotenv-cli is not needed at all for the `import { config } from 'dotenv'` part, but you need to install `dotenv` into your project to be able to do that.